services/repository/transfer_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repository
+
+import (
+	"sync"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/notification"
+	"code.gitea.io/gitea/modules/notification/action"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/unknwon/com"
+)
+
+var notifySync sync.Once
+
+func registerNotifier() {
+	notifySync.Do(func() {
+		notification.RegisterNotifier(action.NewNotifier())
+	})
+}
+
+func TestTransferOwnership(t *testing.T) {
+	registerNotifier()
+
+	assert.NoError(t, models.PrepareTestDatabase())
```

Comment:
@lafriks, This test needs we run `models.MainTest(m, filepath.Join("..", ".."))` before.